content/applications/inventory_and_mrp/purchase/manage_deals/blanket_orders.rst

```diff
@@ -1,204 +1,183 @@
+Blanket orders
+==============
+
+.. _purchase/manage_deals/blanket-orders:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |UoM| replace:: :abbr:`UoM (Unit of Measure)`
+.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
+.. |RfQs| replace:: :abbr:`RfQs (Requests for Quotation)`
 
 Blanket orders are long-term purchase agreements between a company and a vendor to deliver products
-on a recurring basis with predetermined pricing. Using blanket orders are useful when products are
-always purchased from the same vendor, but in different quantities at different times.
+on a recurring basis with predetermined pricing.
+
+Blanket orders are helpful when products are consistently purchased from the same vendor, but in
+different quantities, and at different times.
 
 By simplifying the ordering process, blanket orders not only save time, they also save money, since
 they can be advantageous when negotiating bulk pricing with vendors.
 
-.. seealso::
-   :doc:`calls_for_tenders`
-
 Create a new blanket order
 ==========================
 
-To create blanket orders, the *Purchase Agreements* feature first needs to be enabled in the
-settings of the *Purchase* app. To do this, go to :menuselection:`Purchase --> Configuration -->
-Settings`, and under the :guilabel:`Orders` section, click the checkbox next to :guilabel:`Purchase
-Agreements`. Doing so will enable the ability to create blanket orders, as well as alternative
-requests for quotation (RFQs).
+To create blanket orders, enable the *Purchase Agreements* feature from the *Purchase* app settings.
+Navigate to :menuselection:`Purchase app --> Configuration --> Settings`, and under the
+:guilabel:`Orders` section, click the checkbox for :guilabel:`Purchase Agreements`.
 
-.. image:: blanket_orders/blanket-orders-settings-page.png
+.. note::
+   In addition to creating blanket orders, the *Purchase Agreements* setting also allows users to
+   create alternative requests for quotation (RfQs).
+
+.. image:: blanket_orders/blanket-orders-enabled-setting.png
    :align: center
-   :alt: Purchase Agreements enabled in the Purshase app settings.
+   :alt: Purchase Agreements enabled in the Purchase app settings.
 
-To create a blanket order, go to :menuselection:`Purchase --> Orders --> blanket orders`, and click
-:guilabel:`New`. This creates (and navigates to) a new blanket order form.
+To create a blanket order, go to :menuselection:`Purchase app --> Orders --> Blanket Orders`,
+and click :guilabel:`New`. This opens a blank blanket order form.
 
-From this new blanket order form, different fields and settings can be configured, so there are
-pre-determined rules that the recurring long-term agreement must follow:
+Configure the following fields in the new blanket order form to establish predetermined rules for
+the recurring long-term agreement:
 
 - :guilabel:`Purchase Representative`: is the user assigned to this specific blanket order. By
```

Comment:
I'd suggest removing the "...is" lead-in from all these descriptions, since a colon is being used.

Reply:
Suggestion implemented ✅